src/chart/RechartsWrapper.tsx

```diff
@@ -314,12 +369,20 @@ export const RechartsWrapper = forwardRef<HTMLDivElement | null, RechartsWrapper
       <TooltipPortalContext.Provider value={tooltipPortal}>
         <LegendPortalContext.Provider value={legendPortal}>
           <WrapperDiv
-            width={width}
-            height={height}
+            width={width ?? style?.width}
+            height={height ?? style?.height}
             className={clsx('recharts-wrapper', className)}
             style={{
               position: 'relative',
               cursor: 'default',
+              /*
+               * Some charts may render outside their boundaries,
+               * maybe because of a bug or because of rounding errors.
+               * If that happens, and the chart is responsive, it would keep growing forever
+               * to accommodate the chart that always renders a little bit too large.
+               * Let's have overflow hidden here so that the chart size stabilises.
```

Comment:
So it does actually stabilize? Or it looks like it does

Reply:
With overflow=hidden, the SVG still renders larger than the width it got assigned (X+N) but the RechartWrapper will not fire its own ResizeObserver in reaction to that. The N pixels are hidden.

So in a way this overflow hidden changes from one bug (chart grows forever until the browser crashes) to another bug (N pixels are cut off from the bottom of the chart) which I find acceptable.